app/src/main/java/io/github/droidkaigi/confsched2018/data/repository/ContributorDataRepository.kt

```diff
@@ -33,7 +41,8 @@ class ContributorDataRepository @Inject constructor(
     companion object {
         private const val OWNER = "DroidKaigi"
         private const val REPO = "conference-app-2018"
-        private const val MAX_PER_PAGE = 300
+        private const val MAX_PER_PAGE = 100
+        private const val MAX_PAGE = 2
```

Comment:
Can you comment the meaning of `2` and issue url here? 🙇

Reply:
at first, i tried to get link header and use it's url.
but i could not find good way to do that and link header is not formatted(it means hearder is not json or something like that)
so i used this const value to manage current max page num of contributor...

Reply:
i'm sorry.
you meant to add comment line for this value on source code?

Reply:
Yes, because this number should be updated if the number of contributors will be over 200. I think it would be better to add the comment like this.
// If the number of the contributors will be over 200, this number must be changed to 3.